### PR TITLE
Invariant fix: round up quote tokens calculated from rewarded LP (bec…

### PR DESCRIPTION
**Pull request by grandizzy**
…ause LP rewarded are calculated in bucketTake as rewarded quote tokens -> LP rounded down)

## Tasks

- [ ] Changes to protocol contracts are covered by unit tests executed by CI.
- [ ] Protocol contract size limits have not been exceeded.
- [ ] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [ ] Scope labels have been assigned as appropriate.
- [ ] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

**Review by mattcushman — APPROVED** [review]
lgtm

File: tests/forge/invariants/base/handlers/unbounded/UnboundedLiquidationPoolHandler.sol [hunk 1]
```diff
@@ -408,7 +402,8 @@ abstract contract UnboundedLiquidationPoolHandler is BaseHandler {
     }
 
     // Helper function to calculate quote tokens from lps in a bucket irrespective of deposit available.
-    function lpToQuoteToken(uint256 lps_, uint256 bucketIndex_) internal view returns(uint256 quoteTokens_) {
+    // LP rewarded -> quote token rounded up (as LP rewarded are calculated as rewarded quote token -> LP rounded down)
```
Comment (EdNoepel) [inline tests/forge/invariants/base/handlers/unbounded/UnboundedLiquidationPoolHandler.sol:405]: I don't understand the parenthesized portion of the comment.

**Review by EdNoepel — APPROVED** [review]
LGTM

File: tests/forge/invariants/base/handlers/unbounded/UnboundedLiquidationPoolHandler.sol (hunk 1 above)
Comment (grandizzy) [reply tests/forge/invariants/base/handlers/unbounded/UnboundedLiquidationPoolHandler.sol:405]: I wanted to point out that in `TakerActions` we start from rewarded quote tokens and transform to LP (rounding down) here https://github.com/ajna-finance/contracts/blob/merge-rc9/src/libraries/external/TakerActions.sol#L619

**Pull request merged**